doc/source/serve/batching-guide.md

```diff
@@ -0,0 +1,46 @@
+# Requests Batching
+
+Serve offers a request batching feature that can improve your service throughput with sacrificing latency.
+
+This guide will teach you:
+- How to use Serve's `@serve.batch` decorator
+- How to configure `@serve.batch` decorator
+
+ML frameworks like Tensorflow, PyTorch, and Scikit-Learn are built to support efficiently evaluating multiple samples at the same time.
```

Comment:
Machine Learning (ML) frameworks like Tensorflow, PyTorch, and Scikit-Learn efficiently   support and evaluate multiple samples at the same time.

OR drop the adverb: both are in active voice and strong action verbs.

Machine Learning (ML) frameworks like Tensorflow, PyTorch, and Scikit-Learn support and evaluate multiple samples at the same time.